audit/lambda/s3Services/putS3Object.ts

```diff
@@ -0,0 +1,19 @@
+import { S3Client, PutObjectCommand, PutObjectCommandInput } from '@aws-sdk/client-s3';
+
+export const putS3Object = async (bucket: string, fileKey: string, data: string): Promise<void> => {
+    const client = new S3Client({ region: process.env.AWS_REGION });
+
+    const input = {
+        Bucket: bucket,
+        Key: fileKey,
+        Body: data,
+    } as PutObjectCommandInput;
+
+    console.log(input);
```

Comment:
I think take this console.log out, along with the one below, guess you just had them in for debugging?

Reply:
Again, was for debugging, will take out